File: C Programs/1st Sem MCA/fibonacci.c
// Program to generate fibonnaci series

#include<stdio.h>
#include<stdlib.h>

int fib(int n)
{
	if(n==0)
	{
		return 0;
	}
	else if(n==1)
	{
		return 1;
	}
	else
	{
		return fib(n-1)+fib(n-2);
	}
}

int main()
{
	int n;
	printf("\n-----------------------------------------------------------------------");
	printf("\n                          FIBONACCI SERIES                             ");
	printf("\n                         ------------------                            ");
	printf("\n\n Enter the number of terms : ");
	scanf("%d",&n);
	printf("\n The fibonnaci series is : ");
	for(int i=0;i<n;i++)
	{
		printf("%d ",fib(i));
	}
	printf("\n\n-----------------------------------------------------------------------\n\n");
	return 0;
}

File: Java Programs/3rd Sem BCA/fibonacci.java
import java.util.Scanner;
public class fibonacci
{
 public static void main(String arg[])
 {
 int fib1=0,fib2=1,temp=0,num;
 Scanner sc= new Scanner(System.in);
 System.out.println("Enter the number of terms");
 num=sc.nextInt();
 System.out.println("The fibonacci Series...");
 System.out.println(fib1);
 System.out.println(fib2);
 for (int i=2;i<num;i++)
 {
 temp=fib1+fib2;
 System.out.println(temp);
 fib1=fib2;
 fib2 = temp;
 }
 }
}
